src/interface_adapter/game_settings/GameSettingsViewModel.java

```diff
@@ -0,0 +1,41 @@
+package interface_adapter.game_settings;
+
+import interface_adapter.ViewModel;
+import interface_adapter.menu.MenuState;
+
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
+
+public class GameSettingsViewModel extends ViewModel {
+
+    public final String DIFFICULTY_SELECTOR_LABEL = "Difficulty: ";
+    public final String GENRE_SELECTOR_LABEL = "Genre: ";
+    public final String LIVES_SPINNER_LABEL = "Lives: ";
+    public final String ROUNDS_SPINNER_LABEL = "Rounds: ";
+
+    public final String PLAY_GAME_LABEL = "PLAY GAME";
+    public final String BACK_LABEL = "BACK";
+
+
+    private GameSettingsState state = new GameSettingsState();
+
+    public GameSettingsViewModel() {
+        super("game settings view");
+    }
+
+    @Override
+    public void firePropertyChanged() {
+
```

Comment:
Does this need to be implemented later on?

Reply:
I don't think so? There nothing in GameSettings that needs to change I think. I just put it there because Intellij prompted me. We can keep it there for completeness or just in case.

Reply:
Okay sounds good! Looks good then